Rakefile

```diff
@@ -25,7 +25,7 @@ Dir['tasks/**/*.rake'].each { |t| load t }
 begin
   require 'packaging'
   Pkg::Util::RakeUtils.load_packaging_tasks
-rescue => e
+rescue LoadError => e
   puts "Error loading packaging rake tasks: #{e}"
```

Comment:
Is there a reason the load error isn't fatal?

Reply:
@e-gris yes. When packaging does any work (including just loading these rake tasks), it does a lot of preloading to set params. One of the things it does is use `git describe` to determine the ref/version of the project. When any rake task from this file is run (including tasks totally unrelated to packaging), packaging still does this loading. The `git describe` fails in some cases, like if we're not in a git repo, or if AppVeyor doesn't clone deep enough to get a version. 😜 We still want to be able to run other rake tasks, so this is a workaround. 
I opened https://tickets.puppetlabs.com/browse/RE-11307 to fix this in packaging, but that's gonna a be a significant chunk of work.